images/macos/provision/utils/utils.sh

```diff
@@ -82,6 +84,20 @@ is_Less_BigSur() {
     fi
 }
 
+verify_http_status_code()
+{
+    URL="$1"
+    http_code=$(curl -s -o out.html -w '%{http_code}'  http://www.google.com/linux;)
```

Comment:
Should we replace google to URL?

Reply:
@al-cheb of course, it's just copied example, thank you for pointing to this.